flutter_launcher_icons-development.yaml

```diff
@@ -1,14 +1,14 @@
 flutter_icons:
```

Comment:
Please change this to: `flutter_launcher_icons`. `flutter_icons` is deprecated, so probably why it isn't working.